engineyard/ey-cookbooks-stable-v5
# [FBZ-5075] PHP update

**Description of your patch**
Updates version of PHP to PHP 7.2.34

**Recommended Release Notes**
Updated PHP version.

**Estimated risk**
Low

**Components involved**
Default version of packages

**Description of testing done**
* Built test package
* Deploy PHP application
* Make sure application is accessible and running PHP 7.2.34
